### DISCUSSION TITLE
A good way to remember extension name

### DISCUSSION THREAD

**Discussion by qubitz** [seachicken/gh-poi#97] [Q&A] [answered]
Why the name `gh poi`? An explanation would help me to remember to use this supercool extension.

**Comment by seachicken** [comment, marked as answer]
Hi, "poi" means "feel free to throw it away" in Japanese 🙂
I aim to delete unwanted branches with this command casually.

**Comment by seachicken** [comment]
I added the question to the README. Thanks!
https://github.com/seachicken/gh-poi#why-the-name-poi

**Comment by indigoviolet** [comment]
Consider adding an alias that is more descriptive `clean-branches` -- this will make it easier for people to remember.

**Comment by seachicken** [comment]
Hmm, `clean-branches` is already used. https://github.com/davidraviv/gh-clean-branches
You can set the alias with `gh alias set clean-branches poi` if you like 🙂 https://cli.github.com/manual/gh_alias_set

**Comment by indigoviolet** [comment]
Oh that's great, I'll just do that. Consider adding that to the docs :)

**Comment by seachicken** [comment]
Yeah, it is nice. I added it now.
https://github.com/seachicken/gh-poi/blob/main/README.md#why-the-name-poi
